keras/layers/containers.py

```diff
@@ -75,6 +75,19 @@ def updates(self):
         return updates
 
     @property
+    def state_updates(self):
```

Comment:
A docstring comment on how the system works would be welcome. I get it, but the 2 different behaviors make it quite non-obvious.